DuckDuckGo/AppDelegate.swift

```diff
@@ -82,13 +82,22 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
     
     private func startOnboardingFlowIfNotSeenBefore() {
+        
         var settings = TutorialSettings()
         if !settings.hasSeenOnboarding {
+            startMigration()
```

Comment:
It should be a pretty quick operation, but my worry is that by decoupling it from the on-boarding there may be edge cases where the migration isn't finished by the time the user gets to their bookmarks.  Do we need to manage the UI/UX in this case, or are we OK with that potential edge case?